pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -653,35 +647,35 @@ var _ = Describe("Converter", func() {
 			vmi.ObjectMeta.UID = "f4686d2c-6e8d-4335-b8fd-81bee22f4814"
 		})
 
-		var convertedDomain = strings.TrimSpace(fmt.Sprintf(embedDomainTemplateX86_64, domainType, "%s"))
+		var convertedDomain = strings.TrimSpace(fmt.Sprintf(embedDomainTemplateX86_64, "kvm", "%s"))
```

Comment:
Can we inline `kvm` into the templates?

Reply:
Yes, let me try that. Even though I have bigger fish to fry: unit tests job are passing, but the same unit tests are failing in the goverall job.